### ISSUE TITLE
assert(permutation[0] == 0)  # cannot move batch dim

### ISSUE THREAD

**Issue by Ken1256** [NVIDIA-AI-IOT/torch2trt#807]
https://github.com/cszn/SCUNet/blob/main/models/network_scunet.py

```
if __name__ == '__main__':

    from torch2trt import torch2trt
    
    device = 'cuda'
    net = SCUNet().eval().to(device)
    x = torch.randn((1, 3, 128, 128)).to(device)
    
    net = torch2trt(net, [x])
    
    x = net(x)
    print(x.shape)
```
